packages/babel-plugin-proposal-class-properties/test/fixtures/general/regression-T7364/expected.js

```diff
@@ -42,7 +42,7 @@ class MyClass {
 
 });
 
-export default class MyClass3 {
+class MyClass3 {
```

Comment:
Can we split the class and the export statement only if it is actually needed? (Only if it doesn't add too much complexity to the plugin)

Reply:
It was easier to split those for now in all cases, need to check how much complexity it would add to keep them together (and the same should be done in transform-classes as it is splitting those every time)